extension/src/fileSystem/index.test.ts

```diff
@@ -828,6 +828,42 @@ describe('addPlotToDvcYamlFile', () => {
       mockDvcYamlContent + mockPlotYamlContent
     )
   })
+
+  it('should add a new plot when plot list items have no indent', () => {
+    const mockDvcYamlContent = [
+      'plots:',
+      '- eval/importance.png',
+      '- Precision-Recall:',
+      '    x: recall',
+      '    y:',
+      '        eval/prc/train.json: precision',
+      '        eval/prc/test.json: precision'
+    ].join('\n')
```

Comment:
You can use string literals to test for format (`) instead of creating lines and joining them.

Reply:
Yes, I prefer creating lines since I find it easier to keep track of the exact spacing on each line.